ui/app/models/identity/entity.js

```diff
@@ -43,12 +43,13 @@ export default IdentityModel.extend({
   inheritedGroupIds: attr({
     readOnly: true,
   }),
-
-  updatePath: identityCapabilities(),
   canDelete: alias('updatePath.canDelete'),
   canEdit: alias('updatePath.canUpdate'),
   canRead: alias('updatePath.canRead'),
-
-  aliasPath: lazyCapabilities(apiPath`identity/entity-alias`),
   canAddAlias: alias('aliasPath.canCreate'),
 });
+
+export default attachCapabilities(Model, {
```

Comment:
I don't remember if `attachCapabilities` was already back ported to 1.2.x - something to look out for when you do that.